nips/11.md

```diff
@@ -0,0 +1,61 @@
+NIP-11
+======
+
+Relay Information Document
+---------------------------
+
+`draft` `optional` `author:scsibug`
+
+Relays may provide server metadata to clients to inform them of capabilities, administrative contacts, and various server attributes.  This is made available as a JSON document over HTTP, on the same URI as the relay's websocket.
+
+When a relay receives an HTTP(s) request with an `Accept` header of `application/nostr+json` to a URI supporting WebSocket upgrades, they SHOULD return a document with the following structure.
+
+```json
+["NOSTR_SERVER_INFO",
+{
+  name: <string identifying relay>,
+  descr: <string with detailed information>,
+  pubkey: <administrative contact pubkey>,
+  email: <administrative contact email address>,
+  supported_nips: <a list of NIPS supported by the relay>,
+  software: <string identifying relay software URL>,
+  version: <string version identifier>
+}]
```

Comment:
Can we return just a JSON object instead of this array?

Reply:
Are you saying just a standalone object instead of the array with the NOSTR_SERVER_INFO (or NOSTR_RELAY_INFO)?  I was probably thinking about how I was going to do this as a websocket message, so I agree it would make sense to omit the array container and just return the underlying object.